.github/workflows/basic-ci-amd64.yaml

```diff
@@ -23,11 +23,26 @@ jobs:
       matrix:
         containerd_version: ['lts', 'active']
         vmm: ['clh', 'dragonball', 'qemu', 'stratovirt', 'cloud-hypervisor']
+        use-runtime-rs: ['no']
```

Comment:
This might be confusing for the casual CI user. This `use-runtime-rs` parameter is ignored when VMM is `dragonball` or `cloud-hypervisor`. Also CLH folks had a different approach : a new VMM value was created for runtime-rs. Should we do the same for QEMU ?

Cc @pmores

Reply:
Thanks for the feedback. This means that CLH are testing both runtime and runtime-rs atm, right? Could you share a link for their approach? I will do the same thing.

Reply:
ah.... you meant `clh` and `cloud-hypervisor`? I will have a look at it tomorrow. Thanks! 😉

Reply:
Yes I meant exactly that. Maybe @jodh-intel or @amshinde can provide insights ?